floss/get_go_strings.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+
+import sys
+import struct
+import logging
+import argparse
+from typing import List, Iterable, Optional
+
+import pefile
+
+from floss.results import StaticString, StringEncoding
+
+logger = logging.getLogger(__name__)
+
+MIN_STR_LEN = 4
+
+
+def extract_go_strings(
+    sample: str,
+    min_length,
+) -> Iterable[StaticString]:
+    """
+    Get Go strings from a PE file.
+    Reference: https://github.com/mandiant/flare-floss/issues/779
+    """
+
+    try:
+        pe = pefile.PE(sample)
+    except pefile.PEFormatError as err:
+        logger.debug(f"NOT valid PE header: {err}")
+        sys.exit(0)
```

Comment:
functions in library code should not exit the process, because the caller may have some ability to handle the error. instead, raise an exception, like `ValueError` or a subclass of that.